database/schema/controller.go

```diff
@@ -97,9 +97,8 @@ CREATE TABLE change_log_namespace (
 CREATE UNIQUE INDEX idx_change_log_namespace_namespace
 ON change_log_namespace (namespace);
 
--- TODO (stickupkid): Add the namespaces we want to track.
--- INSERT INTO change_log_namespace VALUES
---    (1, 'foo');
+INSERT INTO change_log_namespace VALUES
+    (1, 'external_controller');
```

Comment:
Although I wrote this, are we sure that we want to run with this? It's harder to debug, although I'm not a fan of putting the name in there because migrating the DDL will be more difficult.

Reply:
Since --build-agent will almost always be a development concern, I'm OK with adding to to the base schema DDL without upgrade migrations _except_ in the case where we have to modify between patch versions.